include/bout/index_derivs_interface.hxx

```diff
@@ -308,9 +308,34 @@ template <typename T>
 T VDDY(const T& vel, const T& f, CELL_LOC outloc = CELL_DEFAULT,
        const std::string& method = "DEFAULT", const std::string& region = "RGN_NOBNDRY") {
   AUTO_TRACE();
-  const bool fHasParallelSlices = (f.hasParallelSlices());
-  const bool velHasParallelSlices = (vel.hasParallelSlices());
-  if (fHasParallelSlices && velHasParallelSlices) {
+
+  auto* localmesh = f.getMesh();
```

Comment:
This is duplicated from the `flowDerivative` method I think, is it worth naming this in a function?

Reply:
There's enough local variables that are needed outside the snippet that I didn't think it worth it. We'd have either calculate them outside the function again, or return them from the function

Reply:
Worth adding a comment to say what this should be consistent with (i.e. the code in the other routine)?